### PR TITLE
[TACHYON-1034] Remove Long cast in getFileld() of listFilesTest since it's not neede…

### PR DESCRIPTION
**Pull request by kelvinchu**
https://tachyon.atlassian.net/browse/TACHYON-1034

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

File: integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java [hunk 1]
```diff
@@ -633,10 +633,10 @@ public void listFilesTest() throws InvalidPathException, FileDoesNotExistExcepti
     List<FileInfo> infoList = mFsMaster.getFileInfoList(mFsMaster.getFileId(new TachyonURI("/")));
     for (FileInfo info : infoList) {
       // TODO(Gene): After info.getFileId return long, remove this type cast.
```
Comment (gpang) [inline integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java]: Can you remove this TODO comment as well? Thanks!

**Comment by gpang** [comment]
Thanks for this fix @kelvinchu ! I just left a minor comment.

**Comment by calvinjia** [comment]
Jenkins, add to whitelist.

**Comment by gpang** [comment]
LGTM

**Pull request merged**